website/src/components/layout/main.tsx

```diff
@@ -1,11 +1,12 @@
 import React, { FC, PropsWithChildren, useEffect, useRef } from "react";
 import { useDispatch } from "react-redux";
 import styled from "styled-components";
-import { hasScrolled } from "../../state/common";
-import { PageTop } from "../misc/page-top";
+
+import { PageTop } from "@/components/misc/page-top";
+import { hasScrolled } from "@/state/common";
 import { Footer } from "./footer";
 
-export const Main: FC<PropsWithChildren> = ({ children }) => {
+export const Main: FC<PropsWithChildren<unknown>> = ({ children }) => {
```

Comment:
why `unknown`?

Reply:
because in the version of typings used (v17) we are forced to pass `<P>` ... while in the future version v18, it defaults to `unknown` if you are not passing it.

Reply:
Yeah, PropsWithChildren isn't even the best way to go and if I remember correct should be replaced.